presto-docs/src/main/sphinx/release/release-0.290.rst

```diff
@@ -0,0 +1,124 @@
+* Add function :func:`ip_prefix_subnets` that splits the input prefix into subnets the size of the new prefix length. :pr:`23656`
+* Add new configuration property ``eager-plan-validation-enabled`` for eager building of validation of a logical plan before queuing. :pr:`23649`
+* Add Presto icon to .idea. :pr:`23579`
+* Add session property ``inline_projections_on_values`` and configuration property ``optimizer.inline-projections-on-values`` to evaluate project node on values node. :pr:`23245`
+* Add support in QueuedStatement protocol to accept pre-minted query id and slug. :pr:`23407`
+* Add support to proxy AuthorizedIdentity using JWT. :pr:`23546`
+* Add support for casting ``char`` datatype to various numeric datatypes. :pr:`23792`
+* Add a new NodeManager ``PluginNodeManager``. :pr:`23863`
+* Replace configuration property ``async-cache-full-persistence-interval`` with ``async-cache-persistence-interval``. :pr:`23626`
+* Remove ``array_dupes`` and ``array_has_dupes`` alias names from functions :func:`array_duplicates` and :func:`array_has_duplicates`. :pr:`23762`
+* Upgrade avro to version 1.11.4. :pr:`23868`
+* Upgrade commons-codec to version 1.17.0. :pr:`23868`
+* Upgrade commons-compress to version 1.26.2. :pr:`23868`
+* Upgrade commons-io to version 2.16.0. :pr:`23794`
+* Upgrade commons-io to version 2.16.1. :pr:`23868`
+* Upgrade commons-lang3 to version 3.14.0. :pr:`23868`
+* Upgrade protobuf-java to version 3.25.5. :pr:`23797`
+* Upgrade protobuf-java-util to version 3.25.5. :pr:`23797`
+* Upgrade to centralize dependency versions in the parent pom.xml from presto-cassandra/pom.xml for consistency and ease of maintenance.  :pr:`23818`
+
+Presto C++ (Native Execution) Changes
+______________________________________
+* Fix ``task.writer-count`` and ``task.partitioned-writer-count`` configuration properties in Presto C++ for consistency with Presto. :pr:`23902`
+* Fix a bug where users weren't able to set the ``native_expression.max_array_size_in_reduce`` session property. :pr:`23856`
+* Fix plan validation failures for some join queries running with spill enabled when using Presto C++. :pr:`23595`
+* Improve I/O and CPU usage by merging FilterNode into TableScanNode where possible. Also, added ``$path`` and ``$bucket`` to split info, and fixed the split counts in the coordinator UI. :pr:`23755`
+* Improve arbitrator configs to use the new string-based format. :pr:`23496`
+* Add test for logical type storage in Parquet files. :pr:`23388`
+* Add a metric ``presto_cpp.memory_pushback_expected_reduction_bytes`` to track expected reduction in memory after a pushback attempt. :pr:`23872`
+* Add a new counter, presto_cpp.memory_pushback_reduction_bytes, to monitor the actual memory reduction achieved with each memory pushback attempt. :pr:`23813`
+* Allow setting chunk size in ChunkedSliceOutput, it can reduce memory in the cases of large number of small output buffer. :pr:`23707`
```

Comment:
```suggestion
* Improve setting buffer chunk size in ChunkedSliceOutput, which can reduce memory in the cases of large number of small output buffers. :pr:`23707`
```
Suggested revision to follow the phrasing in the [Order of changes](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes) of the Release Notes Guidelines. 

I read the original #23707 and I think this is good, but please review for accuracy. 

NOTE: Because of the revision, please move this entry to adjacent to lines 56-57 "Improve I/O.." and "Improve arbitrator...".

Reply:
Allow is better since it cleary conveys what was done